File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/04. Movie Stars/04. Movie Stars.py
all_budget = float(input())

while True:
    try:
        action_name = input()
        if action_name == 'ACTION':
            break

        if len(action_name) > 15:
           all_budget *= 0.80
           continue
        else:
            honorarium = float(input())
            all_budget -= honorarium

        if all_budget <= 0:
            break
    except:
        break


if all_budget >= 0:
    print(f'We are left with {all_budget:.2f} leva.')
else:
    print(f'We need {abs(all_budget):.2f} leva for our actors.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/04. Movie Stars/04. Movie Stars.js
function movieStars(...arg) {
    let allBudget = +arg.shift()

    while (true) {    
            let actionName = arg.shift()
            if (actionName == 'ACTION') {
                break
            }

            if (actionName.length > 15){
                allBudget *= 0.80
            continue
            }else{
                honorarium = +arg.shift()
            }
            allBudget -= honorarium

            if (allBudget <= 0){
                break
            }
        }

    if (allBudget >= 0)
        return `We are left with ${allBudget.toFixed(2)} leva.`
    return `We need ${(Math.abs(allBudget)).toFixed(2)} leva for our actors.`
}

// console.log(movieStars(
//     '90000',
//     'Christian Bale',
//     '70000.50',
//     'Leonard DiCaprio',
//     'Kevin Spacey',
//     '24000.99',
//     'ACTION'))